ts/features/profile/store/actions/index.ts

```diff
@@ -0,0 +1,14 @@
+import { ActionType, createAsyncAction } from "typesafe-actions";
+import { NetworkError } from "../../../../utils/errors";
+import { InitializedProfile } from "../../../../../definitions/backend/InitializedProfile";
+
+/**
+ * The user requests the EU Covid certificate, starting from the auth_code
+ */
+export const getProfile = createAsyncAction(
+  "PROFILE_REQUEST",
+  "PROFILE_SUCCESS",
+  "PROFILE_FAILURE"
```

Comment:
What do you think about modifying this name by adding the action you're performing?
```suggestion
  "PROFILE_GET_REQUEST",
  "PROFILE_GET_SUCCESS",
  "PROFILE_GET_FAILURE"
```